app/src/features/apiClient/screens/apiClient/utils.ts

```diff
@@ -184,3 +185,13 @@ export const convertFlatRecordsToNestedRecords = (records: RQAPI.Record[]) => {
   return updatedRecords;
 };
 export const getEmptyPair = (): KeyValuePair => ({ id: Math.random(), key: "", value: "", isEnabled: true });
+
+export const setToSessionStorage = (key: string, data: string | [] | {}) => {
+  typeof window !== "undefined" && window.sessionStorage.setItem(key, JSON.stringify(data));
+};
+
+export const getFromSessionStorage = (key: string, emptyValue: string | [] | {}) =>
+  (typeof window !== "undefined" &&
+    !isEmpty(window.sessionStorage.getItem(key)) &&
+    JSON.parse(window.sessionStorage.getItem(key))) ||
+  (typeof emptyValue === "undefined" ? {} : emptyValue);
```

Comment:
This util function can be used for other instances or features as well. Lets create a separate util file for sessionStorage actions in `utils` folder at root level

Reply:
We already have a util for localStorage, you can take reference from there.
https://github.com/requestly/requestly/blob/master/app/src/utils/localStorage.ts